plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/panel/grouping/GroupingResultsContainer.java

```diff
@@ -248,9 +263,21 @@ public void rebuildGrouping() throws DBException {
         DataEditorFeatures.RESULT_SET_PANEL_GROUPING.use(Map.of(
             "custom", isCustomQuery,
             "default", isDefaultGrouping,
-            "dups", isShowDuplicatesOnly));
+            "dups", isShowDuplicatesOnly
+        ));
         groupingViewer.setDataFilter(dataFilter, true);
-        //groupingViewer.refresh();
+        // groupingViewer.refresh();
```

Comment:
Commented-out code that will be lost after this PR. Consider removing it if it's redundant